insights/parsers/ifcfg.py

```diff
@@ -126,3 +127,10 @@ def bonding_mode(self):
         if m in bond_mode_map:
             return bond_mode_map[m]
         return None
+
+    @property
+    def has_empty_line(self):
+        """
+        (bool) `True` if the file has empty line else `False`.
+        """
+        return '' in self.context.content
```

Comment:
Could a line which only contains spaces, like `   ` can be treated as a blank line and has the same effect to empty line``?

Once it is. Better check them as well.

`any(l.strip() == '' for l in context)`

Otherwise, please ignore this comment.

Reply:
Could a line which only contains spaces, like can be treated as a blank line and has the same effect to empty line``?
> No